Exam/CC1/src/exercice4/ReservationManageur.java

```diff
@@ -0,0 +1,48 @@
+package exercice4;
+
+import java.util.ArrayList;
+import java.util.Date;
+
+public class ReservationManageur {
+
+	private final ArrayList<Reservation> reservations;
+	
+	public ReservationManageur() {
+		reservations = new ArrayList<>();
+	}
+	
+	public void creerReservation(Reservation b) {
```

Comment:
This works but... this method does not *actually* create a `Reservation`. It just registers a *given* `Reservation`.
You thus force the user of `ReservationManageur` to create himself a Reservation object.
Cf Simple Factory pattern we've seen today.